frontend/src/components/pages/AdminDashboard.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const AdminDashboard = (): React.ReactElement => {
+  return (
+    <div style={{ textAlign: "center", width: "25%", margin: "0px auto" }}>
```

Comment:
maybe we can try a useStyles hook instead of inline styles. It would be more maintainable in the future when we add more styling